package.json

```diff
@@ -14,12 +14,12 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "lint:fix": "eslint --fix .",
     "lint:report": "eslint -f checkstyle -o build2/reports/lint/eslint-checkstyle-result.xml .",
-    "test": "grunt test",
+    "test": "npm run lint && npm run test:unit",
```

Comment:
nit:
usually we didn't combine unit test and lint

Reply:
Good catch - we're already running this script in [Travis](https://github.com/okta/okta-auth-js/blob/master/.travis.yml#L7) and [internally](https://github.com/okta/okta-auth-js/blob/master/scripts/lint.sh#L8). I'll remove it.